requirements.txt

```diff
@@ -1,3 +1,4 @@
 autoroutes==0.2.0
 httptools==0.0.9
+mimetype-match==1.0.4
```

Comment:
Should we have a `requirements_ext.txt`, or should we just document the fact that this package needs be installed for the `content_negotiation` use?
We could easily check in the extension call that the module is installed, and raise if not.